src/config_file.c

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+#include <stdio.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <string.h>
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
+#include "config_file.h"
+#include "lib/sedcli_log.h"
+
+#define SEDCLI_CONF_DELIM "="
+
+#define ARRAY_SIZE(x) (sizeof(x) / sizeof(x[0]))
+
+enum {
+	KMIP_IP = 0,
```

Comment:
By default the first value of an enum is 0.
It shouldn't be necessary to specify this.

Reply:
I guess it is also ok to explicitly set the value of enum instead of rely on implicit defaults. I will go ahead with this patch